### PR TITLE
update workshop with new titan embeddings

### PR DESCRIPTION
**Pull request by lauerarnaud**
*Issue #, if available:*

*Description of changes:*
- Update BedrockEmbeddings to use latest embeddings model v2.
- Tidy up from typos and text, mainly in chatbot notebooks (reference to GPT-J, Chroma, etc).

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by rppth** [comment]
We merged a chatbot update recently, can we resolve the conflict first?

File: 00_Intro/bedrock_boto3_setup.ipynb [hunk 1]
```diff
@@ -632,13 +632,15 @@
     "\n",
     "Use text embeddings to convert text into meaningful vector representations. You input a body of text \n",
     "and the output is a (1 x n) vector. You can use embedding vectors for a wide variety of applications. \n",
-    "Bedrock currently offers one model for text embedding that supports text similarity (finding the \n",
+    "Bedrock currently offers Titan Embeddings for text embedding that supports text similarity (finding the \n",
     "semantic similarity between bodies of text) and text retrieval (such as search).\n",
-    "For the text embeddings model, the input text size is 512 tokens and the output vector length is 4096.\n",
+    "\n",
+    "At the time of writing you can use `amazon.titan-embed-g1-text-02` as embedding model via the API. The input text size is 512 tokens and the output vector length is 4096.\n",
```
Comment (ihmaws) [inline 00_Intro/bedrock_boto3_setup.ipynb:638]: Incorrect input/output size. The v2 model uses 8192 and 1536, respectively.

Also, do you think we should list both the v1 and v2 models?
```suggestion
    "At the time of writing you can use `amazon.titan-embed-g1-text-02` as embedding model via the API. The input text size is 8192 tokens and the output vector length is 1536.\n",
```

Comment (lauerarnaud) [reply 00_Intro/bedrock_boto3_setup.ipynb]: Thanks for feedback. Corrected this.

**Comment by rsgrewal-aws** [comment]
approved

**Review by rsgrewal-aws — APPROVED** [review]
approved

**Review by ihmaws — CHANGES_REQUESTED** [review]
There are still some sections not updated correctly. Will try to highlight them shortly.

**Review by ihmaws — CHANGES_REQUESTED** [review]

File: 03_QuestionAnswering/01_qa_w_rag_claude.ipynb [hunk 2]
```diff
@@ -228,7 +240,9 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
```
[the remark this answers is not in the document]
Comment (lauerarnaud) [reply 03_QuestionAnswering/01_qa_w_rag_claude.ipynb:243]: Fixed that one.

**Comment by rsgrewal-aws** [comment]
have the conflicts been resolved ?

**Comment by rppth** [comment]
This branch has conflicts that must be resolved
Use the [web editor](https://github.com/aws-samples/amazon-bedrock-workshop/pull/39/conflicts) or the to resolve conflicts.
Conflicting files
04_Chatbot/00_Chatbot_Titan.ipynb

**Comment by lauerarnaud** [comment]
It is the kernel. I am using Data Science 3 compared to conda_python_3. It is the same for the other notebooks and wasn't flagged.

**Comment by lauerarnaud** [comment]
Resolved conflict.

**Pull request merged**